design.md

```diff
@@ -0,0 +1,8 @@
+Design which is in the below link is Admin related view.
+
+Design:- https://balsamiq.cloud/s165djm/p41w40c
+
+Flow diagram is the complete figure of the website.
+
+Flow diagram:- https://drive.google.com/file/d/1oNx0eAEpnyqtqz17votFGa7UD9SOK9nl/view?usp=sharing
```

Comment:
@saimanoharhm the design is great. But for Flow chart you can add a image to the markdown file instead of a drive link. Is there any reason for this?

Reply:
> @saimanoharhm the design is great. But for Flow chart you can add a image to the markdown file instead of a drive link. Is there any reason for this?
Not a specific reason. Sure I will upload it as a image.